web_src/js/features/stopwatch.js

```diff
@@ -152,7 +152,7 @@ function updateStopwatchTime(seconds) {
   if (!Number.isFinite(secs)) return;
 
   const start = Date.now();
-  updateTimeInterval = setInterval(() => {
+  return setInterval(() => {
```

Comment:
Good catch, some old code seems incorrect before.

Also suggest line 152 to `return null` for consistency.